UserShiftsApiService/UserShiftsApiService/Services/ISaveNewEmployeeService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+using UserShiftsApiService.Models;
+
+namespace UserShiftsApiService.Services;
+
+public interface ISaveNewEmployeeService
+{
+    public Task SaveNewEmployee(EmployeeItem employee);
```

Comment:
in services, not controllers. any async functions should postfix with `Async` ie `PerformAsync()`

Reply:
functions in interfaces dont need to specify public